app/src/main/java/org/mozilla/reference/browser/BrowserActivity.kt

```diff
@@ -46,7 +49,7 @@ open class BrowserActivity : AppCompatActivity() {
      * Returns a new instance of [BrowserFragment] to display.
      */
     open fun createBrowserFragment(sessionId: String?): Fragment =
```

Comment:
Having `BrowserActivity` call `createBrowserFragment` which would create `MainContainerFragment` seems forced.
RB uses one activity per feature so if the feature now is "Main.."
Maybe this should be renames to `MainActivity` & `createMainFragment`.
Speaking of which, why do we need a fragment parent for home & browser and not just use this Activity?